iron-input.html

```diff
@@ -130,8 +150,9 @@
     },
 
     /**
-     * Returns true if the value is valid. The validator provided in `validator` will be used first,
+     * Returns true if `value` is valid. The validator provided in `validator` will be used first,
      * then any constraints.
+     * @return {Boolean} True if the value is valid.
```

Comment:
Supposedly we take Closure-style type annotations. According to the Closure docs, everything is lowercase (boolean, string, number) except for Object or Array.

https://developers.google.com/closure/compiler/docs/js-for-compiler#types